src/index.postcss

```diff
@@ -1,3 +1,41 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+  .container {
```

Comment:
nie ustaliliśmy w końcu, że jednak to usuwasz? Czy zostawiłeś tylko jakieś konkretne?

Reply:
Tak, zakręciłem się, już usunięte.